src/openrct2/ParkImporter.cpp

```diff
@@ -117,22 +117,14 @@ namespace ParkImporter
 
     bool ExtensionIsRCT1(const std::string &extension)
     {
-        if (String::Equals(extension, ".sc4", true) ||
```

Comment:
I find the old version easier to read.

Reply:
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!